sui_programmability/examples/experiments/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Should this go into `examples/basics`? I'm trying to avoid having too many top-level directories under `examples` if we can. 

If we do want to add a new directory here: 
- `examples/README.md` should be updated with a description of what kind of code can be found in the new dir
- The new dir needs its own `README.md` explaining each example.

Reply:
Yes, it should go somewhere else; even more - it shouldn't be anywhere, I published this thingy by accident. 😝